std-bits/table/src/main/java/org/enso/table/data/column/storage/BoolStorage.java

```diff
@@ -391,4 +398,45 @@ public BoolStorage slice(List<SliceRange> ranges) {
 
     return new BoolStorage(newValues, newMissing, newSize, negated);
   }
+
+  @Override
+  public Storage<?> cast(StorageType targetType, CastProblemBuilder castProblemBuilder) {
+    return switch (targetType) {
+      case AnyObjectType any ->
+        new MixedStorageFacade(this);
+      case BooleanType booleanType ->
+        this;
+      case FloatType floatType -> {
+        int n = size();
+        long[] newData = new long[n];
+        long t = Double.doubleToRawLongBits(1.0);
+        long f = Double.doubleToRawLongBits(0.0);
+        for (int i = 0; i < n; i++) {
+          newData[i] = values.get(i) ? t : f;
+        }
+        yield new DoubleStorage(newData, n, isMissing);
+      }
+      case IntegerType integerType -> {
+        int n = size();
+        long[] newData = new long[n];
+        for (int i = 0; i < n; i++) {
+          newData[i] = values.get(i) ? 1 : 0;
+        }
+        yield new LongStorage(newData, n, isMissing);
+      }
+      case TextType textType -> {
+        int n = size();
+        StringBuilder builder = new StringBuilder(n);
+        for (int i = 0; i < n; i++) {
+          if (isMissing.get(i)) {
+            builder.appendNulls(1);
+          } else {
+            builder.append(values.get(i) ? "true" : "false");
```

Comment:
```suggestion
            builder.append(values.get(i) ? "True" : "False");
```

As that is Enso's default?

Reply:
Yes and no.

In the Table visualization it used to be lowercase due to JSON, right?
Also in the REPL it is a bit inconsistent.
<img width="187" alt="image" src="https://github.com/enso-org/enso/assets/1436948/21c93245-93d5-4aac-867a-dddbeed92b2f">

But ok I can make it Title case, I think it makes sense - but we should probably align places like Table too (unless it was aligned already)